conftest.py

```diff
@@ -60,10 +65,11 @@ def mysql_db_user():
 
 
 @pytest.fixture
-def postgresql_dsn(postgresql_db_user, postgresql_db_password, db_name):
-    return 'postgresql://{0}:{1}@localhost/{2}'.format(
+def postgresql_dsn(postgresql_db_user, postgresql_db_password, postgresql_db_host, db_name):
```

Comment:
This line is causing a lint failure in CI. Would you run `black` on the code and update the PR? It will auto-format this line so linters pass. Thanks!